src/vs/workbench/contrib/extensions/browser/extensionEditor.ts

```diff
@@ -69,6 +69,27 @@ import { insane } from 'vs/base/common/insane/insane';
 
 function removeEmbeddedSVGs(documentContent: string): string {
 	return insane(documentContent, {
+		allowedTags: [
+			'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'h7', 'h8', 'br', 'b', 'i', 'strong', 'em', 'a', 'pre', 'code', 'img', 'tt',
+			'div', 'ins', 'del', 'sup', 'sub', 'p', 'ol', 'ul', 'table', 'thead', 'tbody', 'tfoot', 'blockquote', 'dl', 'dt',
+			'dd', 'kbd', 'q', 'samp', 'var', 'hr', 'ruby', 'rt', 'rp', 'li', 'tr', 'td', 'th', 's', 'strike', 'summary', 'details',
+			'caption', 'figure', 'figcaption', 'abbr', 'bdo', 'cite', 'dfn', 'mark', 'small', 'span', 'time', 'wbr'
+		],
+		allowedAttributes: {
+			'*': [
+				'accesskey', 'action', 'align', 'alt', 'aria-describedby', 'aria-hidden', 'aria-label', 'aria-labelledby',
```

Comment:
For now, let's just add `align` here. I don't know that the online marketplace supports all of these attributes, and we'd want to align with them. We can look into supporting additional attributes if people request them

Reply:
Changed